test/Microsoft.ML.AutoML.Tests/AutoFitTests.cs

```diff
@@ -183,16 +183,16 @@ public void AutoFitRegressionTest(string culture)
                 // Furthermore, these issues might only occur after ~70
                 // iterations, so setting the internal maxModels parameter.
                 int maxModels = culture == "en-US" ? 1 : 75;
+                var context = new MLContext(1);
```

Comment:
Can these test changes (and the ones in the next file) be reverted? Looks like they aren't necessary anymore.

Reply:
I believe so. I'll revert them and test it.

Reply:
Reverted.